Xamarin.Forms.Core/ToastArguments.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace Xamarin.Forms.Internals
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public class ToastArguments
+	{
+		public ToastArguments(string message, int duration, string actionButtonText, Func<Task> action)
+		{
+			Duration = duration;
+			Message = message;
+			ActionButtonText = actionButtonText;
+			Action = action;
+			Result = new TaskCompletionSource<bool>(false);
+		}
+
+		/// <summary>
```

Comment:
We can drop these inline comments

Reply:
just want to be consistent with similar file (AlertArguments as example)